lib/OpenQA/Utils.pm

```diff
@@ -816,6 +816,11 @@ sub read_test_modules {
 
         for my $step (@{$module->details}) {
             $step->{num} = $num++;
+            if ($step->{text}) {
+                log_error("Should read information in ");
```

Comment:
I don't get the purpose of logging an error

Reply:
It was just for me to debug :) Removing